products/sentinel_one.py

```diff
@@ -71,17 +124,18 @@ def __init__(self, profile: str, creds_file: str, account_id: Optional[list[str]
         account_names = account_name if account_name else list()
 
         # extract account/site ID from configuration if set
-        if 'account_id' in config[profile] and config[profile]['account_id'] not in account_ids:
-            account_ids.append(config[profile]['account_id'])
+        if 'account_id' in config[self.profile] and config[self.profile]['account_id'] not in account_ids:
+            account_ids.append(config[self.profile]['account_id'])
```

Comment:
```suggestion
        if 'account_id' in config[self.profile]:
            for id in config[self.profile]['account_id'].split(','):
                if id not in account_ids:
                    account_ids.append(id)
```
add a check similar to this in order to allow multiple account_ids to be specified in a config file.